superset/config.py

```diff
@@ -732,6 +732,10 @@ class CeleryConfig:  # pylint: disable=too-few-public-methods
             "task": "reports.prune_log",
             "schedule": crontab(minute=0, hour=0),
         },
+        "key_value.cleanup": {
+            "task": "key_value.cleanup",
+            "schedule": timedelta(seconds=10),
```

Comment:
How long does it take to run the jobs? 10 seconds looks a bit aggressive to me, maybe we could have a default of ~1 minute to be conservative on resources?

Reply:
I forgot to change that 🤪. I was just using 10 seconds for testing but the default should be 24 hours.